src/cartesian/ticks/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { CartesianTickItem } from '../../util/types';
+
+/**
+ * Given an array and a number N, return a new array which contains every nTh
+ * element of the input array. For n below 1, an empty array is returned.
+ * @param {any[]} array An input array.
```

Comment:
nit: comment says `any[]` but it has a generic type (even if that type can be anything, much better than `any`). Same two lines down.

Reply:
There might even be support for generic types in jsdoc. https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template